ProcessMaker/processmaker
# FOUR-9704

## Issue & Reproduction Steps
Some information needs correction

## How to Test

1. Review the following events:
  a. Publish a Template
  b. Update a script
2. Check the User Activity Logging 

## Related Tickets & Packages
[- Link to any related FOUR tickets, PRDs, or package](https://processmaker.atlassian.net/browse/FOUR-9704)s

## Code Review Checklist
